.github/workflows/track-prs-for-release.yml

```diff
@@ -36,6 +36,7 @@ jobs:
         env:
           CASPER_TOKEN: ${{ secrets.CASPER_TOKEN }}
           GITHUB_TOKEN: ${{ steps.generate_token.outputs.token }}
+          LOGS_URL: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
```

Comment:
The Github context should be available at the top level, so I'm pretty sure this duplication could be eliminated by moving it up from the step to the job or workflow level.